.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@edlynhaynes @carl-ritchie @Grayson @AllanGuigou @daniel-cook
\ No newline at end of file
+* @edlynhaynes @carl-ritchie @Grayson @AllanGuigou @daniel-cook
```

Comment:
What does the * do? Is that just part of the standard formatting for this file?

Reply:
* assigns all files in the repo to the defined codeowners; alternatively we could specify another set of codeowners for a specific folder, etc, instead of using *.

Reply:
Oh cool, TIL 👍